package.json

```diff
@@ -7,6 +7,8 @@
     "name": "Ain Tohvri",
     "email": "ain@flashbit.net"
   },
+  "main": "dist/smartbanner.min.js",
+  "style": "dist/smartbanner.min.css",
```

Comment:
For the reasons described here https://stackoverflow.com/a/48673965/1373324 I'd not point to a minified versions of the distribution. If someone needs them though, they'd still be there.

Reply:
Changed them to be non minified.